Gruntfile.js

```diff
@@ -120,7 +112,7 @@ module.exports = function(grunt) {
             '!config_room_wallsetter.js',
             '!config_string.js'
           ],
-          dest: 'dist/',
```

Comment:
I have no preference. I understand the reasoning for both.
I like to have it consistent and I like it to be checked automatically. Maybe it is easy to check via jshint / jscs, add it to the config and we are fine.